lib/ui/src/modules/ui/components/left_panel/header.js

```diff
@@ -16,8 +15,9 @@ const headingStyle = {
   fontSize: '12px',
   fontWeight: 'bolder',
   color: '#828282',
+  textAlign: 'center',
   cursor: 'pointer',
-  padding: 0,
+  padding: '5px',
   margin: 0,
   overflow: 'hidden',
```

Comment:
I feel like `text-overflow: ellipsis` could be helpful so it wouldn't break into multiple lines?

Reply:
I think it's actually better when it wraps, as it did before: 
<img width="188" alt="screen shot 2017-10-04 at 01 23 21" src="https://user-images.githubusercontent.com/6651625/31193226-30669aae-a94c-11e7-9d9d-b71f10d9e18e.png">